collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
For 1 - So would you take teams out of collector_item and put them into their own repo?
For 2 - I don't think issue data should be separated out by team / project since there would be a lot of duplication. There should be a team repo (listing all teams), a project repo (listing all projects), and a feature repo (having all issues). Features then have IDs that relate them to their team / project / neither
For 3 - I don't agree. Both can be 'Any' to display everything. These are simply filters into the feature repo. There could be more filters in the future so we wouldn't want to segregate data but instead keep them in feature
For 4 - Sounds fair but may be a lot of work. I assume all features will still be shared?